packages/driver/cypress/integration/e2e/multi-domain/commands/multi_domain_route.spec.ts

```diff
@@ -0,0 +1,18 @@
+// @ts-ignore / session support is needed for visiting about:blank between tests
```

Comment:
There is a spec for `route` and `server` that is currently being skipped if you wanted to update it.

https://github.com/cypress-io/cypress/blob/feature-multidomain/packages/driver/cypress/integration/e2e/multi-domain/commands/multi_domain_network_requests.spec.ts#L31

Reply:
Totally forgot about those. Going to go ahead an removed the skipped test

Reply:
Also looks like there are some issues with with `clearCookies` tests which I need to take a look at.

Reply:
What about merging this and `multi_domain_server.spec.ts` into something like `multi_domain_unsupported_apis.spec.ts`? Seems silly to have a whole file for one test and then if we decide not to support any other commands, they could be added to it as well.

Reply:
I'm good with that. I originally had something like that but wondered if it would deviate too much from the pattern. I'll condense them!

Reply:
So little `cy` cookie commands rely on big `Cypress` to clear cookies. For the meantime, I figured we can unsupported `preserveOnce` in multi domain and revisit this decision in the near future.